docs/references/account/create-magic-url-session.md

```diff
@@ -1,3 +1,3 @@
-Sends the user an email with a secret key for creating a session. If the provided user ID has not be registered, a new user will be created. When the user clicks the link in the email, the user is redirected back to the URL you provided with the secret key and userId values attached to the URL query string. Use the query string parameters to submit a request to the [PUT /account/sessions/magic-url](/docs/client/account#accountUpdateMagicURLSession) endpoint to complete the login process. The link sent to the user's email address is valid for 1 hour. If you are on a mobile device you can leave the URL parameter empty, so that the login completion will be handled by your Appwrite instance by default.
```

Comment:
Hmmm, some of these are actually not correct.

This should still be `accountUpdateMagicURLSession`.

`accountUpdateMagicURLSession` completes the authentication flow.

`accountCreateMagicURLSession` starts the authentication flow.

Please look through your PR and make sure the updates are context appropriate!

Reply:
There was also the same issue with PhoneSession. I have updated it.